spanner/google/cloud/spanner_admin_database_v1/types.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+import sys
+
+from google.gax.utils.messages import get_messages
+
+from google.api import auth_pb2
+from google.api import http_pb2
+from google.cloud.spanner_admin_database_v1.proto import spanner_database_admin_pb2
+from google.iam.v1 import iam_policy_pb2
+from google.iam.v1 import policy_pb2
+from google.iam.v1.logging import audit_data_pb2
+from google.longrunning import operations_pb2
+from google.protobuf import any_pb2
+from google.protobuf import descriptor_pb2
+from google.protobuf import empty_pb2
+from google.protobuf import timestamp_pb2
+from google.rpc import status_pb2
+
+names = []
+for module in (
+        auth_pb2,
+        http_pb2,
+        spanner_database_admin_pb2,
+        iam_policy_pb2,
+        policy_pb2,
+        audit_data_pb2,
+        operations_pb2,
+        any_pb2,
+        descriptor_pb2,
+        empty_pb2,
+        timestamp_pb2,
+        status_pb2, ):
+    for name, message in get_messages(module).items():
+        message.__module__ = 'google.cloud.spanner_admin_database_v1.types'
```

Comment:
I don't think we ever resolved my concern about hiding the real source package for the proto-generated message types.

Reply:
That is right, we did not.

The goal here is that I want the idiom for users to be `from google.cloud.api_vN import types`, and then we promise that all of the types that they need exist there. I feel like it is a pain for users to have to care about `random_modulename_pb2`, which I believe is surprising (also, if you are a user, what the heck does `_pb2` even mean?).

This approach means that the documentation documents _all_ of these messages under `types`, so that is all a user has to care about, unless he goes and spelunks the code (at which point the import chain is explicit).

Reply:
My memory is that we were not publishing API docs for the `__init__` module, but this PR does add them for the package and for `types`, which ameliorates the problem.

Have we examined the generated docs for completeness / clarity in the context of the move?

Reply:
They are effectively just API reference and types. No narrative docs. :-(
I would welcome narrative docs but do not have time to write them, and they are not a requirement.